sechub-scan/src/main/java/com/mercedesbenz/sechub/domain/scan/admin/FullScanDataRestController.java

```diff
@@ -62,7 +62,7 @@ public void getFullScanZipFileForJob(
             support.writeScanData(fullScanData, outputStream);
         } catch (IOException e) {
             LOG.error("Was not able to provide zip file for full scan data of {}", logSanitizer.sanitize(sechubJobUUID, -1), e);
-            throw new NotFoundException("Was not able to support zip file, see logs for details");
+            throw new NotFoundException("Was not able to networkTargetDataSupport zip file, see logs for details");
```

Comment:
What do you mean? What does `networkTargetDataSupport` and `zip file` have in common?

Reply:
This was accidently changed by wrong refactoring setup in IDE ("change all occurrences + renamed "support" field  to "networkTargetDataSupport" .

This is not the only occurrence!

I will change this for all accidently renamed parts.